.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: Node.js Package
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - run: cd core
+    - run: yarn install
+    - run: yarn publish --patch
```

Comment:
Why patch?

Reply:
I don't know :( it should be either a patch or a minor, but really it is a human decision right?